server/graphql/v2/query/TransactionsQuery.ts

```diff
@@ -78,7 +78,7 @@ const TransactionsQuery = {
       where['FromCollectiveId'] = fromAccount.id;
     }
     if (account) {
-      where['CollectiveId'] = account.id;
+      where[Op.or] = [{ CollectiveId: account.id }, { UsingVirtualCardFromCollectiveId: account.id, type: 'DEBIT' }];
```

Comment:
I think this condition will be overwritten in https://github.com/opencollective/opencollective-api/blob/d1a6c9d959906517d18acb03c879fdd0f02f21fb/server/graphql/v2/query/TransactionsQuery.ts#L86 if we provide a `searchTerm`.